src/EventHandlers.js

```diff
@@ -15,12 +28,14 @@ let { AaveOracleContract, AvalancheAaveOracleContract, PolygonAaveOracleContract
 //   let awesomeEventObject = context.awesomeEvent.get(event.params.identifier);
 //   context.awesomeEvent.set(awesomeEventObject);
 // });
-AaveOracleContract.AssetSourceUpdated.loader = (event, context) => {
-  context.AssetSourceUpdated.load(AssetSourceUpdated);
-};
+AaveOracleContract.AssetSourceUpdated.loader((event, context) => {
```

Comment:
This is the most important change that I made. 

Both the loader & handler are functions that take in a function. 

ie loader((event, context) = {})
not loader = (event, context) => {}